event_handler.go

```diff
@@ -41,6 +41,19 @@ func eventHandler(ctx *IrcContext, rtm *slack.RTM) {
 				if err != nil {
 					log.Printf("Error getting channel info for %v: %v", ev.Msg.Channel, err)
 					channame = "unknown"
+				} else if channel.IsMpIM {
+					channame = "#mpdm-" + ev.Msg.Channel
```

Comment:
What about using a different prefix symbol instead of `#mpdm-`? It will make it less likely to have collisions. E.g. the channel name could be `@channame` instead of `#mpdm-channame`. The `@` symbol is not used anywhere as far as I know, and it would be no different from the IRC networks that use `#` or `&` to prefix channels

Reply:
besides - I plan to use `&` for secret channels, in a similar way to weeslack